keyboards/controllerworks/city42/info.json

```
@@ -0,0 +1,133 @@
+{
+    "manufacturer": "Controller Works",
+    "keyboard_name": "city42",
+    "maintainer": "controller-works",
+    "processor": "RP2040",
+    "url": "https://controller.works/products/city42-ergonomic-keyboard",
+    "tags": ["RP2040", "choc v1", "choc spaced" ],
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0005",
+        "vid": "0x4357"
+    },
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "build": {
+        "lto":true
+    },
+    "matrix_pins": {
+        "cols": 
+            ["GP28", "GP27", "GP26","GP25", "GP24", "GP23", "GP0", "GP1", "GP2", "GP3", "GP4", "GP5"],
+        "rows":
+            ["GP12", "GP13", "GP14", "GP15"]
```

keeps in uniform with everybody else 😉
```suggestion
        "cols": ["GP28", "GP27", "GP26","GP25", "GP24", "GP23", "GP0", "GP1", "GP2", "GP3", "GP4", "GP5"],
        "rows": ["GP12", "GP13", "GP14", "GP15"]
```